e2e/various-suite/geomap-map-controls.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { e2e } from '@grafana/e2e';
+const DASHBOARD_ID = 'P2jR04WVk';
+
+e2e.scenario({
+  describeName: 'Geomap map controls options',
+  itName: 'Tests map controls options',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.openDashboard({ uid: DASHBOARD_ID, queryParams: { editPanel: 1 } });
+
+    // Show zoom
+    e2e.components.OptionsGroup.group('Map controls').scrollIntoView().should('exist');
+    e2e.components.PanelEditor.toggleShowZoom().check({ force: true });
+    cy.contains('+');
+    cy.get('.ol-zoom').should('exist');
+
+    // Mouse wheel zoom
+    e2e.components.OptionsGroup.group('Map controls').should('exist');
+    e2e.components.PanelEditor.toggleMouseWheelZoom().check({ force: true });
+    cy.get('.panel-content').trigger('wheel', {
+      deltaY: -66.666666,
+      wheelDelta: 120,
+      wheelDeltaX: 0,
+      wheelDeltaY: 120,
+      bubbles: true,
+    });
```

Comment:
Does firing this event when mouse zoom is disabled break the test? Not sure how straightforward it will be to provide a solid test case for mouse zoom / ensuring zoom functionality is working as expected via an e2e test (other than the recorded screencast)